### PR TITLE
op-mode: T5884: Correction of description for generate-wireguard.xml.in

### PR DESCRIPTION
**Pull request by aga1314**
## Change Summary

Correct the 'generate wireguard' command help string

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [x] Other (please describe): Wrong description (probably from copy-paste)

## Related Task(s)

* https://vyos.dev/T5884

## Component(s) name

OP-Mode-Description of generate-wireguard.xml.in 

## Proposed changes

The description "Generate WireGuard keys" doesn't belong to this command. It should be updated to something fitting like "Generate WireGuard client config QR code"

## How to test

Running "generate ?" in op-mode:
```
user@VyOS:~$ generate ?
Possible completions:
  container             Generate Container Image
  firewall              Firewall
  interfaces            Interface specific commands
  ipsec                 Generate IPsec related configurations and archives
  macsec                Generate MACsec Key
  openconnect           Generate OpenConnect client parameters
  openvpn               Generate OpenVPN client configuration ovpn file
  pki                   Generate PKI certificates and keys
  public-key-command    Generate configuration mode command to add OpenSSH public key
                        from file
  ssh                   Generate SSH related keypairs
  system                Generate system related parameters
  tech-support          Generate tech support info
  wireguard             Generate WireGuard keys
```

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: op-mode-definitions/generate-wireguard.xml.in [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <children>
       <node name="wireguard">
         <properties>
-          <help>Generate WireGuard keys</help>
+          <help>Generate WireGuard Client config QR code</help>
```
Comment (c-po) [inline op-mode-definitions/generate-wireguard.xml.in]: Should be improved to: `Generate WireGuard client configuration and QR code`

**Review by c-po — CHANGES_REQUESTED** [review]

**Comment by sever-sever** [comment]
Check commit message. Expected format with the task number

File: op-mode-definitions/generate-wireguard.xml.in (hunk 1 above)
Comment (dmbaturin) [inline op-mode-definitions/generate-wireguard.xml.in]: Why capitalize "Client"? It's not a proper noun.

**Review by dmbaturin — CHANGES_REQUESTED** [review]
Please come up with an informative commit message, too, like "correct the 'generate wireguard' command help string" at least.

File: op-mode-definitions/generate-wireguard.xml.in (hunk 1 above)
Comment (aga1314) [reply op-mode-definitions/generate-wireguard.xml.in]: Sorry, I just accepted the requested change..

**Comment by c-po** [comment]
Squashed and updated commit message

**Pull request merged**